docs/03-tutorials/00-eventing/evnt-04-change-max-in-flight-in-sub.md

```diff
@@ -39,29 +39,40 @@ The "in-flight messages" config defines the number of events that Kyma Eventing
    
    ```bash
    cat <<EOF | kubectl apply -f -
-     apiVersion: serverless.kyma-project.io/v1alpha1
-     kind: Function
-     metadata:
-       labels:
-         serverless.kyma-project.io/build-resources-preset: local-dev
-         serverless.kyma-project.io/function-resources-preset: S
-         serverless.kyma-project.io/replicas-preset: S
-       name: lastorder
-       namespace: default
-     spec:
-       deps: '{ "dependencies": {}}'
-       maxReplicas: 1
-       minReplicas: 1
-       source: |
-         module.exports = {
-           main: async function (event, context) {
-             console.log("Processing event:", event.data);
+   apiVersion: serverless.kyma-project.io/v1alpha2
+   kind: Function
+   metadata:
+     labels:
+       serverless.kyma-project.io/build-resources-preset: local-dev
```

Comment:
Now we rather prefer these settings inside `resourceConfiguration` except `serverless.kyma-project.io/replicas-preset`. In general I have the same notices like for `docs/03-tutorials/00-eventing/evnt-03-type-cleanup.md`